jellyfin-api/src/commonMain/kotlin/org/jellyfin/sdk/api/sockets/SubscriptionExtensions.kt

```diff
@@ -0,0 +1,67 @@
+package org.jellyfin.sdk.api.sockets
+
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.filter
+import org.jellyfin.sdk.model.api.GeneralCommandMessage
+import org.jellyfin.sdk.model.api.GeneralCommandType
+import org.jellyfin.sdk.model.api.OutboundWebSocketMessage
+import org.jellyfin.sdk.model.api.PlaystateCommand
+import org.jellyfin.sdk.model.api.PlaystateMessage
+import org.jellyfin.sdk.model.api.SendCommandType
+import org.jellyfin.sdk.model.api.SyncPlayCommandMessage
+
+/**
+ * Subscribe to a specific WebSocket message type.
+ */
+public inline fun <reified T : OutboundWebSocketMessage> SocketApi.subscribe(): Flow<T> = subscribe(T::class)
+
+/**
+ * Subscribe to specific [GeneralCommandType] messages.
+ */
+@Suppress("NOTHING_TO_INLINE")
+public inline fun SocketApi.subscribeGeneralCommands(
+	commands: Set<GeneralCommandType> = GeneralCommandType.values().toSet(),
+): Flow<GeneralCommandMessage> = subscribe<GeneralCommandMessage>()
+	.filter { message -> message.data?.name in commands }
```

Comment:
I personally prefer this style in this case, but both would work.
```suggestion
): Flow<GeneralCommandMessage> = subscribe<GeneralCommandMessage>().filter { message ->
	message.data?.name in commands
}
```

Reply:
I find each "action" on its own line more readable

Reply:
Alright :+1: